tensorflow/examples/speech_commands/train.py

```diff
@@ -79,6 +79,7 @@
 
 import input_data
 import models
+from six.moves import xrange  # pylint: disable=redefined-builtin
```

Comment:
Actually, can you move this to line 78 (after `import numpy as np`)? The ordering is: builtin modules, third-party modules, project-specific modules. `six` is a third-party module.